### DISCUSSION TITLE
夜神模拟器多开助手无法识别

### DISCUSSION THREAD

**Discussion by kyonyan** [MaaAssistantArknights/MaaAssistantArknights#1755] [Q&A]
帮一位因为工作原因无法自己玩但又不想退坑的朋友刷体力，想用夜神模拟器的多开助手登录另一个方舟账号用脚本帮他刷。但似乎脚本只能链接到主模拟器，新开的无法识别到，有什么解决方案吗？

**Comment by Chengrui2022** [comment]
在连接设置里面有一个“连接地址”，就是127.0.0.1:62***的地址，在夜神模拟器那边找一下教程，用powershell找到另外一个模拟器的连接地址，再复制一份maa开就可以了
